Libraries/Utilities/createStrictShapeTypeChecker.js

```diff
@@ -47,9 +48,12 @@ function createStrictShapeTypeChecker(
     for (var key in allKeys) {
       var checker = shapeTypes[key];
       if (!checker) {
+        const stack =  parseErrorStack(new Error());
+        const renderFrame = stack.find((frame) => frame.methodName.endsWith('render'));
         invariant(
           false,
           `Invalid props.${propName} key \`${key}\` supplied to \`${componentName}\`.` +
+            `\nStack: ${JSON.stringify(renderFrame)}` +
```

Comment:
Also, I somehow only just now realized that this whole thing only applies to `createStrictShapeTypeChecker`, not all warnings - can't you add `const stack =  parseErrorStack(new Error());` inside `updateWarningMap` so we can always have the file/line?

Also, this is in invariant - doesn't it throw a redbox with a stack trace for you?